modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,50 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which can be configured by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
+
+.Prerequisites
+
+* You must have the {ServerlessOperatorName} and Knative Serving installed on your {product-title} cluster.
+
+.Procedure
+
+. Create a Knative service that contains the label or annotation that you want to propagate to the {product-title} route:
++
+.Example service YAML
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+  labels:
+    <label_name>: <label_value>
+  annotations:
+    <annotation_name>: <annotation_value>
+...
+----
++
+.Example kn command
+[source,terminal]
+----
+$ kn service create <service_name> \
+  --image=<image> \
+  --annotation <annotation_name>=<annotation_value> \
+  --label <label_value>=<label_value>
+----
+
+. Verify that the {product-title} route has been created with annotation or label that you added, by inspecting the output from the following command:
```

Comment:
```suggestion
. Verify that the {product-title} route has been created with the annotation or label that you added by inspecting the output from the following command:
```

Reply:
There's not really a good way to show it yet because we don't have a specific example of an annotation yet, the docs are generic on purpose. I'd say we could consider adding this if we go to flesh out the docs later and add more specific examples; I had tried to do this initially, but have been asked by QE not to because it was adding confusion for the moment.